packages/docs-page/server/consts.ts

```diff
@@ -0,0 +1,10 @@
+export const ENABLE_VERSIONED_DOCS = process.env.ENABLE_VERSIONED_DOCS
+export const VERCEL_ENV = process.env.VERCEL_ENV
```

Comment:
nit: I'm not super crazy about re-assigning env vars in a separate file like this. I find it harder to grok for usage and it's just another layer of indirection.

That said, I'm not willing to die on this hill and if you feel this is a better structure I'm okay with that. 😄

Reply:
I'm not tied to this! Happy to revert this change...

I'm more interested in lifting up environment variable usage into function args, so that the functions can be [more pure](https://en.wikipedia.org/wiki/Pure_function). Didn't get to this part yet though...

Reply:
Something like 

```diff
- import { ENABLE_VERSIONED_DOCS, VERCEL_ENV, DEFAULT_PARAM_ID } from './consts'
+ import { DEFAULT_PARAM_ID } from './consts' 
```

Then in a function

```diff
- function generateStaticPaths(context) { ... }

+ const options = {
+  VERCEL_ENV: process.env.VERCEL_ENV,
+  ENABLE_VERSIONED_DOCS: process.env.ENABLE_VERSIONED_DOCS,
+ }
+ 
+ function generateStaticPaths(context, { VERCEL_ENV, ENABLE_VERSIONED_DOCS } = options) { ... }
```

Reply:
Updating this now... and removing the env vars from `consts.ts`